packages/rest/test/acceptance/bootstrapping/application.acceptance.ts

```diff
@@ -61,12 +66,14 @@ describe('Bootstrapping the application', () => {
       }
 
       class MyComponent {
-        controllers = [MyController];
+        controllers: Array<Constructor<{}>> = [MyController];
       }
 
       const app = new Application({components: [MyComponent]});
+      const server = new RestServer(app);
+      app.server(server);
 
-      expect(app.find('controllers.*').map(b => b.key))
+      expect(server.find('controllers.*').map(b => b.key))
```

Comment:
Similarly here - controllers are defined at application level, we should not need any server instance to iterate them.